hack/tools/release/notes.go

```diff
@@ -444,6 +444,16 @@ func modifyEntryTitle(title string, prefixes []string) string {
 	return strings.ToUpper(string(entryWithoutTag[0])) + entryWithoutTag[1:]
 }
 
+// trimAreaFromTitle removes area from title to avoid duplication.
+func trimAreaFromTitle(title, area string) string {
```

Comment:
A few unit test cases would be probably good

Reply:
Sure! would be back with them in a day...

Reply:
Sure no rush!